Code/JavaWrappers/RDKFuncs_doc.i

```diff
@@ -167,7 +169,7 @@ the number of electrons
 */
 public";
 
-%javamethodmodifiers RDKit::MolOps::findPotentialStereoBonds 	( 	ROMol &  	mol, 		bool  	cleanIt = false	  	) 			"
+%javamethodmodifiers RDKit::MolOps::findPotentialStereoBonds 	( 	ROMol &  	mol, 		bool  	cleanIt = false	  	, 		UINT_VECT *  	possibleStereobonds = 0	  	) 			"
```

Comment:
It's better (though not strictly necessary) if the default value of `possibleStereobonds` is `NULL` instead of `0`